app/uk/gov/hmrc/residencenilratebandcalculator/connectors/RnrbConnector.scala

```diff
@@ -31,4 +34,16 @@ class RnrbConnector @Inject()(http: WSHttp) extends ServicesConfig {
 
   def getHelloWorld = http.GET(s"$serviceUrl/residence-nil-rate-band-calculator/hello-world")
   def getStyleGuide = http.GET(s"$serviceUrl/residence-nil-rate-band-calculator/style-guide")
+
+  def send(json: JsValue): Future[Either[String, CalculationResult]] =
+    http.POST(s"$serviceUrl/residence-nil-rate-band-calculator/calculate", json, Seq(("Content-Type", "application/json")))
```

Comment:
Can we move the "residence-nil-rate-band-calculator" segment to a config value somewhere?  It'll be used on any endpoint calls, and it may change down the line.